### PR TITLE
adding document

### PR DESCRIPTION
**Pull request by aksh-h**

### CODE REVIEW

File: docs/GitHubActionsDemo/GitHubActionsDemo.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+In this lab, you will learn about how to configure GitHub Action, Environment Variables and Secrets. This helps you to configure your staging and production environemnt using Azure Resource Manager Template and deploye the application.
+
+# Prerequisites
+1. GitHub account
+1. Azure DevOps Organization
+1. Azure Subscription to deploy resources
+
+# Exercise
+
+1. Create a project in your azure DevOps Organization using [Azuer DevOps Demo Generator](http://azuredevopsdemogenerator-core.azurewebsites.net/?name=actiondemo)
```
Comment (Damovisa) [inline docs/GitHubActionsDemo/GitHubActionsDemo.md:12]: Typo - "Azure"

File: docs/GitHubActionsDemo/GitHubActionsDemo.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+4. Navigate to Build and Deploy YAML file, scroll down to see the environment variables ```RESOURCE_GROUP_NAME``` and  ```RESOURCE_GROUP_LOCATION```. If you wan to change resource group name and resource group location, you can update the values here.
+
+    ![variables](Images/EnvVariables.png)
+
+5. Navigate to Settings and to the Secrets, you should see the following secrets configured under Repository Secrets section
+    
+    ![secrets](Images/Secrets.png)
+
+6. Generate Azure Credentials using RBAC commands. Please refer the document [here](https://github.com/marketplace/actions/azure-login#configure-deployment-credentials) 
+7. ```AZURE_SQL_SERVER_NAME``` and ```WEBSITE_NAME``` will be having preconfigured value, update the Azure Credentials in ```AZURE_CREDENTIALS``` secret
```
Comment (Damovisa) [inline docs/GitHubActionsDemo/GitHubActionsDemo.md:29]: Update
~will be having preconfigured value~ to
will have preconfigured values

**Review by Damovisa — CHANGES_REQUESTED** [review]
Great with those small typos :)

File: docs/GitHubActionsDemo/GitHubActionsDemo.md (hunk 1 above)
Comment (aksh-h) [reply docs/GitHubActionsDemo/GitHubActionsDemo.md]: updated

File: docs/GitHubActionsDemo/GitHubActionsDemo.md (hunk 2 above)
Comment (aksh-h) [reply docs/GitHubActionsDemo/GitHubActionsDemo.md]: updated

**Pull request merged**